templates/kubernetes/docs/1.16/upgrading.md

```diff
@@ -217,13 +214,13 @@ juju config kubernetes-master channel=1.10/stable
 
 If you wanted to try a release candidate for 1.12, the channel would be `1.12/candidate`.
 
-<div class="p-notification--caution">
-  <p markdown="1" class="p-notification__response">
-    <span class="p-notification__status">Note:</span>
-Once the configuration has been changed, the charms will be put into a `blocked` state.
-You must continue the upgrade process, even if you revert the configuration to the
-currently active version of Kubernetes.
-  </p>
+<div class="p-notification--caution is-inline">
+  <div markdown="1" class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <div class="p-notification__message">Once the configuration has been changed, the charms will be put into a `blocked` state.
```

Comment:
```suggestion
    <div class="p-notification__message">Once the configuration has been changed, the charms will be put into a `blocked` state.</div>
```
The rest of the page was being wrapped in this notification

Reply:
Oops!